low-level/actuators/impl/generic/node_hw.py

```diff
@@ -163,7 +165,12 @@ def _process_fru_request(self, node_request):
     def _create_node_fru_json_message(self, specifics, resource_id):
         """Creates JSON response to be sent out to Node Controller Message
            Handler for further validation"""
-        resource_type = "node:fru:{0}".format(self.fru_node_request)
+        resource_type = "node:hw:{0}".format(self.fru_node_request)
+        is_fru, fru_type_unit = self.ipmi_client.is_fru(
+            self.fru_node_request)
+        fru = False
+        if is_fru:
+            fru = str(is_fru) + "/" + fru_type_unit
```

Comment:
lets use : as delimeter, like we have in other fields like resource_type

Reply:
done